telcell/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from telcell.models.dummy import DummyModel
+from telcell.models.model import Model
```

Comment:
Mss handig om hier even een comment in te voegen waarom we hier alleen imports hebben en wat iemand die modellen toe gaat voegen daar mee zou moeten.

(dan wel ooit een chille auto-loader maken hier :clown_face:)